packages/TorneloScoresheet/src/chessEngine/chessTsChessEngine.ts

```diff
@@ -99,30 +127,33 @@ const gameToPeiceArray = (game: Chess): ChessBoardPositions => {
   const board: BoardPosition[][] = [];
 
   // for each row
-  game.board().map((gameRow, rowIdx) => {
-    const row: BoardPosition[] = [];
-
-    // for each col in row
-    gameRow.map((peice, colIdx) => {
-      // no piece
-      if (peice === null) {
-        row.push({
-          piece: null,
-          position: boardIndexToPosition(rowIdx, colIdx),
-        });
-      }
-
-      // existing piece
-      else {
-        row.push({
-          piece: mapEnginePeice(peice),
-          position: boardIndexToPosition(rowIdx, colIdx),
-        });
-      }
-    });
+  game
+    .board()
+    .reverse()
+    .map((gameRow, colIdx) => {
```

Comment:
Since we're not making a mapping transform here, I think it would be more semantic to call `forEach`. Unless you want to transform this into a mapping transform, which might be better? i.e. rather than:

```ts
const board = [];

game.forEach((el) => board.push(fn(el)))
```

You would do:
```ts
const board = game.map(el => fn(el))
```